helm/conjur-config-cluster-prep/generated/conjur-config-cluster-prep.yaml

```diff
@@ -13,7 +13,7 @@ metadata:
     app.kubernetes.io/instance: "conjur-serviceaccount"
     app.kubernetes.io/part-of: "conjur-config"
     conjur.org/name: "conjur-serviceaccount"
-    helm.sh/chart: conjur-config-cluster-prep-0.1.0
+    helm.sh/chart: conjur-config-cluster-prep-0.1.1
```

Comment:
At some point, I think we should change the manifest generation scripts so that the `helm.sh/chart` Annotations are deleted from the generated raw manifests? This Annotation should only be there if the resulting deployments were managed/deployed via that Helm chart, IIUC.

Reply:
Isn't that what the `app.kubernetes.io/managed-by=Helm` annotation is for? Furthermore isn't it helpful to have the include versions of the charts that these raw manifests were generated from?

Reply:
I'll follow up with you about this next week.